DataFormats/simulation/include/SimulationDataFormat/MCCompLabel.h

```diff
@@ -0,0 +1,97 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_MCCOMPLABEL_H
+#define ALICEO2_MCCOMPLABEL_H
+
+#include <Rtypes.h>
+#include <iomanip>
+#include <ios>
+#include <iostream>
```

Comment:
can we avoid having this expensive include in a header? consider moving the print function to .cxx

Reply:
ok